crates/holochain/src/conductor/handle.rs

```diff
@@ -404,13 +404,15 @@ pub struct ConductorHandleImpl<DS: DnaStore + 'static> {
 impl<DS: DnaStore + 'static> ConductorHandleT for ConductorHandleImpl<DS> {
     /// Check that shutdown has not been called
     async fn check_running(&self) -> ConductorResult<()> {
+        tracing::trace!("ConductorHandle::check_running");
```

Comment:
I don't think we want these trace events. It would be better to use the instrument macro to create a span and the put events where you need

Reply:
agreed, reverted the commits that introduced them